dist/select/select.css

```diff
@@ -57,17 +57,19 @@ span.select {
 .select select::-ms-expand {
   display: none;
 }
-.select select[disabled] {
-  background-color: var(--select-disabled-background-color, var(--color-background-secondary));
-  color: var(--select-disabled-foregound-color, var(--color-foreground-ghost));
-}
-.select--borderless select[disabled] {
-  background-color: var(--select-borderless-disabled-background-color, var(--color-background-secondary));
-  color: var(--select-borderless-disabled-foregound-color, var(--color-foreground-disabled));
-}
-.select select[readonly] {
-  background-color: var(--select-disabled-background-color, var(--color-background-secondary));
-  color: var(--select-readonly-foregound-color, var(--color-foreground-ghost));
+.select select[disabled],
+.select--borderless select[disabled],
+.select select[readonly],
```

Comment:
I don't think `readonly` is impacted by the opacity issue. We should just isolate this to what was impacted - `disabled`.

Also, simply using the module name as the selector is sufficient since alternative views via modifiers (`.select--borderless`) are only additive. So for the opacity issue, simply having a separate rule just targeting the component is probably sufficient:

`````css
.select select[disabled] {
  opacity: 1;
}
`````

Reply:
You can verify it in the browser to make sure the one rule gets applied to the alternate versions as well.

Reply:
Ok, fixed. I'm not sure if I like that `readonly` and `disabled` each mostly have the same code that is written twice, but if it is the standard I am ok with it.

Reply:
Looks good. I think it's fine. Some repetition is ok. If these had larger sections of duplication, we'd probably want to create a mixing in the file and reuse the styles instead of duplicating.